formats/polus-tabular-to-feather-plugin/src/main.py

```diff
@@ -0,0 +1,201 @@
+from bfio.bfio import BioReader, BioWriter
+from pathlib import Path
+import fcsparser
+import os
+import argparse
+import logging
+import vaex
+import shutil
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from multiprocessing import cpu_count
+import tqdm
+
+# Import environment variables
+POLUS_LOG = getattr(logging,os.environ.get('POLUS_LOG','INFO'))
+POLUS_EXT = os.environ.get('POLUS_EXT','.ome.tif')
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("main")
+logger.setLevel(POLUS_LOG)
+
+#Set number of processors for scalability
+NUM_CPUS = max(1, cpu_count() // 2)
+
+def csv_to_df(file):
+    """Convert csv into datafram or hdf5 file. Copied partially from Nicholas-Schaub polus-csv-to-feather-plugin
```

Comment:
You can remove the part about copying from the original plugin. Usually this is what I would like you to do, but in this case it can be removed.